teachopencadd/talktorials/T038_protein_ligand_interaction_prediction/talktorial.ipynb

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
<ul><li>Describe interaction a bit more. Is it only about binding? If so maybe mention classical approaches to the same problem.</li><li>For example (missing comma)</li><li>You could motivate the model with virtual screening of compound libraries, for example</li><li>TBC in the end.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>